DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: DominoDataR
 Title: 'Domino Data R SDK'
-Version: 0.2.3
+Version: 0.2.5
```

Comment:
Why skip 0.2.4?

Reply:
I didn't skip it, it was released and I forgot to update.. Wondering if I should jump to 0.2.5 or keep 0.2.4 and re-release on github.